Source/Core/Core/IOS/USB/Emulated/Skylander.h

```diff
@@ -19,7 +19,36 @@ constexpr u8 MAX_SKYLANDERS = 16;
 
 namespace IOS::HLE::USB
 {
-extern const std::map<const std::pair<const u16, const u16>, const char*> list_skylanders;
+enum class Game
+{
+  SpyrosAdv,
+  Giants,
+  SwapForce,
+  TrapTeam,
+  Superchargers,
+  Other,
+};
+enum class Element
+{
+  Magic,
+  Fire,
+  AIR,
```

Comment:
You forgot to turn `AIR` into `Air`.

Reply:
Good catch. I also removed a hanging semicolon in `SkylanderPortalWindow.cpp`